components/brave_rewards/resources/page/components/settingsPage.tsx

```diff
@@ -363,6 +370,12 @@ class SettingsPage extends React.Component<Props, State> {
       this.actions.onSettingSave('contributionMonthly', amount)
     }
 
+    const onVerifyClick = () => {
+      if (externalWallet && externalWallet.verifyUrl) {
```

Comment:
nit: as a defense in depth strategy what do you think about validating if the `verifyURL` has a `bitFlyer` or `uphold` prefix?

Reply:
For Uphold that would be feasible, but for bitFlyer their staging login domain is something similar to `demo<bunch_of_random_characters>.azurewebsites.net` (compiled in with a build-time setting).

Reply:
do you know if the team is planning to transition to a first-party domain for staging, shouldn't be hard to setup?